app/controllers/treatment_reports_controller.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
```

Comment:
I think it would be useful to throw up a :success flash message on create and update.